src/core/probe/manager.rs

```diff
@@ -32,6 +32,7 @@ pub(crate) struct ProbeManager {
     #[cfg(not(test))]
     config_map: libbpf_rs::Map,
 
+    options: Vec<ProbeOptions>,
```

Comment:
Document the member?

Reply:
ack. This fell through the cracks, thanks!